pkgs/tools/misc/go-ios/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "go-ios";
+  version = "1.0.114";
+
+  src = fetchFromGitHub {
+    owner = "danielpaulus";
+    repo = "go-ios";
+    rev = "v${version}";
+    sha256 = "sha256-tOaoZkbLObJJb2oTeCE1JowZ0QySKlpHXCiplGVlCMw=";
+  };
+
+  vendorSha256 = "sha256-GtXnjK3tAEIj4l0HeXYjdJ4MWq824euKkfiy/vA76Sw=";
+
+  postPatch = ''
+    # do not build this (does not work)
+    rm -rf restapi
+
+    # remove these tests, as they fail
+    find . -name '*_integration_test.go' -delete
+    rm ios/testmanagerd/xcuitestrunner_test.go
+  '';
+
+  postInstall = ''
+    # aligns the binary with what is expected from go-ios
+    mv $out/bin/go-ios $out/bin/ios
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "This is an operating system independent implementation of iOS device features. You can run UI tests, launch or kill apps, install apps etc. with it.";
```

E.g.

```suggestion
    description = "An operating system independent implementation of iOS device features";
```

See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description and https://github.com/NixOS/nixpkgs/blob/0138c0f6edc6cc5cdc2f3894a1adc3e0ace4ce1a/CONTRIBUTING.md#submitting-changes